prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -223,3 +223,26 @@ periodics:
         requests:
           memory: 1Gi
           cpu: 400m
+- name: kyma-gke-nightly
+  cron: "0 0 * * 1-5" # At 00:00 on every day-of-week from Monday through Friday
+  labels:
+    preset-slack-notifications: "true"
+    <<: *gke_job_labels_template
+  decorate: true
+  extra_refs:
```

Comment:
this is a periodic job, do we have kyma-project/kyma cloned somehow?

Reply:
My copy&paste error, adding kyma refs :)